pom.xml

```diff
@@ -478,6 +486,10 @@
       <plugin>
         <artifactId>maven-javadoc-plugin</artifactId>
       </plugin>
+      <plugin>
```

Comment:
Why again this plugin tag is needed?

Reply:
If this plugin declaration is removed then the whole thing breaks apart. It makes sure the `exec` plugin is available in the Reactor. A more thorough explanation can be found at [this link](http://andresalmiray.com/multi-module-project-builds-with-maven-and-gradle/).